src/index.ts

```diff
@@ -1,9 +1,10 @@
-import { ApolloServer } from 'apollo-server';
+import { ApolloServer, ApolloError } from 'apollo-server';
+import { GraphQLError, GraphQLFormattedError } from 'graphql';
 import { types as typeDefs, resolvers } from './graphql';
-import { prisma, Option, errorName, formatErr } from './utils';
+import { prisma, Option } from './utils';
+import { v4 } from 'uuid';
```

Comment:
Are you using the linter?

You must split up your dependencies from native dependencies

```suggestion
import { ApolloServer, ApolloError } from 'apollo-server';
import { GraphQLError, GraphQLFormattedError } from 'graphql';
import { v4 } from 'uuid';

import { types as typeDefs, resolvers } from './graphql';
import { prisma, Option } from './utils';
```